commands/sandbox/delete.ts

```diff
@@ -195,6 +203,7 @@ export async function handler(
       logger.error(
         i18n(`${i18nKey}.failure.invalidKey`, {
           account: uiAccountDescription(parentAccountId),
+          authCommand: uiCommandReference('hs auth personalaccesskey'),
```

Comment:
I think this would just be `hs auth` since personalaccesskey is the default (and it gets passed in via option if you want to manually set it)